### ISSUE TITLE
Improve screen resolution

### ISSUE THREAD

**Issue by rvdleun** [rvdleun/onoffice#3]
**Goal**

To improve the screen display in the virtual environment

**Description**

When compared to other solutions(Virtual Desktop, Bigscreen), the displays are still very pixely. When the display is large in the virtual environment, you can work with it, but it isn't as nice as it should be.

Unfortunately, I really have very little experience with this and would love to learn of a way on how to make the screens more readable.

**Deliverables**

* Virtual screens that are more easy on the eyes

**Comment by rvdleun** [comment]
I spent some time yesterday upgrading as many of the framework and libraries to newer versions. Next to improving the performance of the application overall, there has been noticeable increase in quality of the displays. 

Whereas I first needed to increase the screen size to really be able to read a terminal, I could now keep it at the default size to work with it. Switching back the old version definitely made it clear that there is an increase of quality here. I assume that the upgrade from Electron 1.6.2 to 4.2.6 helped out here.

First time I noticed was when I tried it out on the Daydream. Originally, the screen mostly looked like a garbled mix of pixels, but now I can now actually make out the display. But there is a real noticeable difference on the Quest as well.

If I have the time, I will investigate this some more to see if more can be done to improve the quality of the display, but I'm already pretty ecstatic about this.
